pyproject.toml

```diff
@@ -0,0 +1,21 @@
+# 
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/c-code
```

Comment:
This part is surprising to me, especially given the `template = "pure-python"` in .meta.toml.

Reply:
The file was copied from another project, I fixed it.